### PR TITLE
Contributed Project Fuse source code

### PR DESCRIPTION
**Pull request by trcox**
Signed-off-by: Tyler Cox <tyler_cox@dell.com>

### CODE REVIEW

File: jobs/java-maven-template.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Library and Micro Service build creation
+
+# Template for all Java libraries
+- job-template:
+    name: '{name}-library'
+    display-name: '{name}-library'
+    description: '{description}
+
+'
+    project-type: maven
```
Comment (tykeal) [inline jobs/java-maven-template.yaml:10]: We do not allow 'project-type: maven' in our build environment. The templates must use a freestyle type.

File: jenkins_jobs.ini [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[jenkins]
```
Comment (tykeal) [inline jenkins_jobs.ini:1]: This entire file needs to go away. We provide an example jenkins.ini file and our .gitignore is configure to ignore jenkins.ini files themselves as they hold credentials.

File: jobs/global-defaults.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+- defaults:
```
Comment (tykeal) [inline jobs/global-defaults.yaml:1]: We already provide a defaults file and his one runs afoul of several problems we've generally worked around

**Review by tykeal — CHANGES_REQUESTED** [review]
Hmm... I started going through this and marking things up but realized that there's a lot wrong with it.

First you have files that hold credentials. We never post do this, you're going to have to do a squash commit to your PR to get rid of them as well as remove them from the history. Besides, none of those credentials are valid in our environment.

Second, all the job definitions are in jobs directory instead of the JJB directory that is already part of the project. We only pull job definitions out of that directory

Third, all of these jobs are pointing at the bitbucket repositories.

Fourth, I don't see any verify / merge job distinctions here. Pretty much all of the jobs presently don't hold with our standard ideas of how the code flow should work:

PR triggers a verification job
Merge triggers a merge job which produces artifacts

Fifth, the jobs are all defined in one file / location and not broken up a more granular fashion. We generally recommend that projects (or specific project groups) use the following structure for their jobs:

jjb/<project>/<project>.yaml
jjb/<project>/<project-additions>.yaml

Given the number of problems I'm seeing I'll see if I can work up something that is more in line with our normal flows. Going off in a direction that is significantly different from our other supported projects makes it very hard for the LF Release Engineering team to support the jobs.

**Comment by JPWKU** [comment]
recheck

**Comment by JPWKU** [comment]
recheck

**Comment by JPWKU** [comment]
recheck

**Pull request closed without merging**